src/pages/AddHabit.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import HabitForm from "../components/HabitForm";
+import { addHabit } from "../actions/habits";
+import { connect } from "react-redux";
+import Header from "../components/Header";
+
+const AddHabit = props => (
+  <div>
+    <Header />
+    <div className="habits-form">
+      <HabitForm
+        onSubmit={habit => {
```

Comment:
I like that we're adding it to the redux store, but can we also add it to the database via the API in this pull request?

endpoint to do that is `POST /api/habits`